katas/content/marking_oracles/index.md

```diff
@@ -56,6 +56,51 @@ This kata continues the exploration of quantum oracles started in the Oracles ka
     ]
 })
 
+@[exercise]({
+    "id": "marking_oracles__palindrome",
+    "title": "Palindrome",
```

Comment:
I'm aiming to use titles that describe the function the oracle implements, so a slightly rephrased version will work better
```suggestion
    "title": "Palindrome Checker",
```